plop_templates/test_template.txt

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import sinon from 'sinon';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
```

Comment:
maybe add the import statement for the component? `import {{properCase name }} from {{type}}/{{properCase name}}`...or maybe not all components will have this specific path? just a thought